azkaban-web-server/src/main/java/azkaban/webapp/servlet/LoginAbstractAzkabanServlet.java

```diff
@@ -136,7 +136,8 @@ protected void doGet(final HttpServletRequest req, final HttpServletResponse res
   /**
    * Log out request - the format should be close to Apache access log format
    */
-  private void logRequest(final HttpServletRequest req, final Session session) {
+  private void logRequest(final HttpServletRequest req, final Session session,
+      final Boolean allowedQueryString) {
```

Comment:
I didn't know there was a difference 😄 haha. I'll switch it to boolean because that looks like it'll do the job. Thanks!